workers/social/lib/social/models/invitation.coffee

```diff
@@ -25,7 +25,7 @@ module.exports = class JInvitation extends jraphical.Module
   @set
     permissions     :
       'send invitations'  : ['member']
-      'remove invitation' : ['admin']
+      'remove invitation' : ['member']
```

Comment:
@cihangir  As you said `remove permit from function call signature`
Are we able to call static instance that doesn't have permit from client side?
if not, how to make request for user who is member and the invitation is belong to her?